lib/common.sh

```diff
@@ -187,6 +187,9 @@ TEST_MAX_TIME="${TEST_MAX_TIME:-240}"
 FAILS=0
 RESULT_STR=""
 
+# Avoid printing skipped Ansible tasks
+export ANSIBLE_DISPLAY_SKIPPED_HOSTS=no
```

Comment:
I am not sure if it should be set to `no` but `false` maybe?

Reply:
You can check [here](https://docs.ansible.com/ansible/latest/plugins/callback/skippy.html) that it is no. And also if you compare the CI output of this PR and master you will notice that skipped tasks aren't printed anymore.